javascript/organizing_your_javascript_code/classes.md

```diff
@@ -10,19 +10,20 @@ After this lesson and completing the assignments, you will be able to:
 
 - Describe the pros and cons of using classes in JavaScript.
 - Briefly discuss how JavaScript's object creation differs from a language like Java or Ruby.
-- Explain the differences between using a class to define a constructor and other prototype methods.
+- Explain the differences between an object constructor and a class.
 - Explain what "getters" & "setters" are.
```

Comment:
```suggestion
- Explain what "getters" and "setters" are.
```
Not related to your changes, but I think this doesn't hurt to improve while we're here